dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/dynomite/DynomiteStandardTuner.java

```diff
@@ -123,6 +125,7 @@ public void writeAllProperties(String yamlLocation) throws IOException {
                 logger.error("Hashtag must be of length 2. Provided hashtag: " + floridaConfig.getDynomiteHashtag()
                         + " has length: " + floridaConfig.getDynomiteHashtag().length());
                 logger.error("Not setting any hashtag");
+                throw new RuntimeException("Hashtag is larger than 2 characters");
```

Comment:
RuntimeException("Hashtag %s should be exactly 2 characters", floridaConfig.getDynmiteHashtag());